director/director_ui.go

```diff
@@ -48,6 +48,7 @@ type (
 		Longitude    float64                   `json:"longitude"`
 		Writes       bool                      `json:"enableWrite"`
 		DirectReads  bool                      `json:"enableFallbackRead"`
+		Listings     bool                      `json:"enableListings"`
```

Comment:
Match the fact the other flags use the singular.

```suggestion
		Listings     bool                      `json:"enableListing"`
```